portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the comment/suggestion from the form.
+    String textValue = request.getParameter("comment-suggestion");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println(textValue);
+
+    // Say thank you to the user for their contribution
+    response.setContentType("text/html;");
+    
+    // Not currently implemented but would like to acknowledge user
+    //  for submitting a comment
```

Comment:
Optional: One way that we write things to do in industry is to do
`// TODO(bugNumber): This is what still needs to be done for feature X.`
or
`// TODO(personName/username): This is another thing that needs to be done for feature X.`

Using this method means you can just search for `TODO` in your code instead of having to actually read every comment

Up to you if you want to update this for your portfolio!